### PR TITLE
Add codeowner

### PR DESCRIPTION
**Pull request by kg86**
This PR set the code owner to this repository. The code owner is automatically assigned as a reviewer for all PRs.
See also [About code owners - GitHub Docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).

### CODE REVIEW

**Review by kampersanda — APPROVED** [review]
Also, I suggest to set up `Require review from Code Owners` in the main protection rule.

スクリーンショット 2022-11-04 14 24 22

**Comment by kg86** [comment]
@kampersanda Thank you. I enabled the option.

**Pull request merged**
